src/jobs/treeherder_resultset.js

```diff
@@ -64,19 +82,15 @@ export default class TreeherderResultsetJob extends Base {
       return;
     }
 
-    if (this.config.try.enabled && tryProject) {
-      if (
-        tryProject.contains &&
-        lastRev.comment.indexOf(tryProject.contains) === -1
-      ) {
-        console.log(
-            `Skipping submitting graph for project '${repo.alias}'. ` +
-            `Commit does not contain ${tryProject.contains}`
-        );
-        return;
-      }
-      console.log(`Scheduling taskcluster jobs for project '${repo.alias}'`);
-      await this.scheduleTaskGraphJob(resultset, repo, pushref);
+    if (tryProject.contains && lastRev.comment.includes(tryProject.contains)) {
+      console.log(
+          `Skipping submitting graph for project '${repo.alias}'. ` +
+          `Commit does not contain ${tryProject.contains}`
```

Comment:
This should be fixed in 61faa7b4a3c3363e873ece38a5f8aae4e876e283 

Thank you for catching that!